### ISSUE TITLE
Mechanize error when signing up a user

### ISSUE THREAD

**Issue by MaxPleaner** [bolthar/ruby-hackernews#28]
Trying to sign up a user

```ruby
2.2.3 :001 > require 'ruby-hackernews'
true
2.2.3 :002 > include RubyHackernews
Object < BasicObject
2.2.3 :003 > user = User.new("masxplea")
#<RubyHackernews::User:0x0000000429d098 @name="masxplea">
2.2.3 :004 > user.signup("password")
NoMethodError: undefined method `u=' for #<Mechanize::Form:0x00000004448b68>
	from /home/max/.rvm/gems/ruby-2.2.3/gems/mechanize-2.7.3/lib/mechanize/form.rb:217:in `method_missing'
	from /home/max/.rvm/gems/ruby-2.2.3/gems/ruby-hackernews-1.3.7/lib/ruby-hackernews/services/signup_service.rb:12:in `signup'
	from /home/max/.rvm/gems/ruby-2.2.3/gems/ruby-hackernews-1.3.7/lib/ruby-hackernews/domain/user.rb:17:in `signup'
	from (irb):4
	from /home/max/.rvm/rubies/ruby-2.2.3/bin/irb:11:in `<main>'
2.2.3 :005 > 
```

Given the nature of Mechanize, I'm guessing there might have been some change to their login form and the headless navigation code hasn't been updated.

**Comment by bolthar** [comment]
I did not look at the signup any longer after they've added a captcha - but now the captcha is gone. I'll fix it tonight.

**Comment by Entalpi** [comment]
Status on this? The same problem exist when trying to login with a user.

**Comment by bolthar** [comment]
Sorry, I forgot about this issue :-1: 

I fixed it now, if you update the gem to 1.4.0 you should be able to login/signup as usual. Remember that signups are limited on the same IP.
